_includes/table_widget.html

```diff
@@ -0,0 +1,33 @@
+
+<div id="table-main">
+ <div >Download table as: [ <a href="/registry/ontologies.yml">YAML</a> | <a href="/registry/ontologies.jsonld">JSON-LD</a> | <a href="/registry/ontologies.ttl">RDF/Turtle</a>  ]
```

Comment:
html linting? there's an errant space here

Reply:
Thanks but I need this merged 2 years ago.. In the next round!